docs/content/patterns/multi-task-form/index.mdx

```diff
@@ -0,0 +1,271 @@
+
+- **Sequential** - If tasks need to be completed in order, one after the other, use the sequential variant of task list. This variant numbers each task to help indicate that tasks need to be completed in order, starting with task 1.
+- **Non-sequential** - If a form can be completed in any order, use the non-sequential variant of task list seen below. This gives users the flexibility of completing tasks at they please. If they’re not ready or able to complete a certain task, they can move onto another one without getting blocked.
+
+```jsx live
+() => {
+	const [isModalOpen, openModal, closeModal] = useTernaryState(false);
+
+	return (
+		<>
+			<Flex gap={3} flexDirection="column">
+				<TaskList
+					items={[
+						{
+							href: '#',
+							label: 'Personal details',
+							status: 'done',
+							message: 'Short description of the task',
+						},
+						{
+							href: '#',
+							label: 'Address details',
+							status: 'doing',
+							message: 'Short description of the task',
+						},
+						{
+							href: '#',
+							label: 'Upload files',
+							status: 'todo',
+							message: 'Short description of the task',
+						},
+						{
+							href: '#',
+							label: 'Confirm and submit',
+							status: 'todo',
+							message: 'Short description of the task',
+						},
+					]}
+				/>
+				<Flex gap={1}>
+					<Button onClick={() => {}} variant="secondary">
+						Save and exit
+					</Button>
+					<Button onClick={openModal} variant="tertiary">
+						Cancel
+					</Button>
+				</Flex>
+			</Flex>
+			<Modal
+				isOpen={isModalOpen}
+				onDismiss={closeModal}
+				title="Discard form?"
+				actions={
+					<ButtonGroup>
+						<Button onClick={closeModal}>Discard form</Button>
+						<Button variant="secondary" onClick={closeModal}>
+							Save and exit
+						</Button>
+					</ButtonGroup>
+				}
+			>
+				<Text as="p">
+					You will lose all information you've entered. This can't be undone.
+					Alternatively, you can save your form progress and complete it later.
+				</Text>
+			</Modal>
+		</>
+	);
+};
+```
+
+Include 2 actions below the task list:
+
+- **Save and exit** - Allows users to leave the form and save their progress. Form inputs should be saved on exit without validating them. Make sure the user can come back to continue their form at a later time. The user should land on the task list when they come back to continue their form. This helps them understand where they are up to in the process.
+- **Cancel** - Allows users to discard the form. If they have entered any information, make sure you trigger a destructive modal dialog (seen below) to let users know that they will loose their progress.
+
+<Box paddingTop={3}>
+	{/* When PR review is comlete use this image instead
+	<img
+		src="/img/patterns/task-list-destructive-dialog.png"
+		alt="Return to partial completion task"
+	/>
+	*/}
+	<img
+		src="/pr-preview/pr-1585/img/patterns/task-list-destructive-dialog.png"
```

Comment:
This needs to be changed to `"/img/patterns/task-list-destructive-dialog.png"` before merging

Reply:
Not sure if you have already, but we run all images throurh https://tinypng.com/